requirements/requirements.txt

```diff
@@ -12,3 +12,5 @@ torch>=1.11
 safetensors
 flash_attn>=2.0
 einops
+texttable
+auto-gptq
```

Comment:
Can it be an optional requirements? When users want to use it, we can check whether we can import it correctly, if not we can raise an error.

Reply:
FYI: https://github.com/hpcaitech/ColossalAI/blob/d20dceb9a3d1bdcb2376201220f49fca7c7c1be9/colossalai/kernel/cuda_native/mha/mha.py#L13